integration_tests/followme_target.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <cmath>
+#include <thread>
+#include <chrono>
+#include "integration_test_helper.h"
+#include "global_include.h"
+#include "dronecore.h"
+
+using namespace dronecore;
+
+TEST_F(SitlTest, FollowMeTargetTest)
+{
+    DroneCore dc;
+
+    DroneCore::ConnectionResult ret = dc.add_udp_connection();
+    ASSERT_EQ(DroneCore::ConnectionResult::SUCCESS, ret);
+
+    // Wait for device to connect via heartbeat.
+    std::this_thread::sleep_for(std::chrono::seconds(2));
+    Device &device = dc.device();
+
+    while (!device.telemetry().health_all_ok()) {
+        std::cout << "waiting for device to be ready" << std::endl;
+        std::this_thread::sleep_for(std::chrono::seconds(1));
+    }
+
+    Action::Result action_ret = device.action().arm();
+    ASSERT_EQ(Action::Result::SUCCESS, action_ret);
+
+    action_ret = device.action().takeoff();
+    ASSERT_EQ(Action::Result::SUCCESS, action_ret);
+
+    std::this_thread::sleep_for(std::chrono::seconds(5));
+
+    FollowMe::Result followme_result = device.followme().start();
```

Comment:
Why is it `followme` but the files are called `follow_me`? Can you make it consistent?

Reply:
I think, it is taking this name from classname. It won't be proper if I name plugin class name as `Follow_Me`. So, I felt `follow_me` is intuitive than followme.

Reply:
Yer, I need to fix it.